### ISSUE TITLE
Support fence_ events

### ISSUE THREAD

**Issue by dsodmanWork** [catapult-project/catapult#4482]
Add a fence_parser to support fence_ events

**Comment by benshayden** [comment]
@lpy Can I ask you to follow up with @dsodmanWork to see what he needs and guide him through the contribution process?

**Comment by lpy** [comment]
Dave is a Googler, does he need to sign anything?

**Comment by benshayden** [comment]
I don't think so. I just couldn't tell from the issue description whether he might need help navigation the code base or the code review process. If you're already working with him, then carry on. :-)
